src/funcTest/groovy/info/solidsoft/gradle/pitest/functional/AbstractPitestFunctionalSpec.groovy

```diff
@@ -1,17 +1,19 @@
 package info.solidsoft.gradle.pitest.functional
 
+import groovy.transform.CompileDynamic
 import nebula.test.IntegrationSpec
 import nebula.test.functional.ExecutionResult
 
-abstract class AbstractPitestFunctionalSpec extends IntegrationSpec {
+@CompileDynamic
+class AbstractPitestFunctionalSpec extends IntegrationSpec {
 
     void setup() {
         fork = true //to make stdout assertion work with Gradle 2.x - http://forums.gradle.org/gradle/topics/unable-to-catch-stdout-stderr-when-using-tooling-api-i-gradle-2-x#reply_15357743
         memorySafeMode = true   //shutdown Daemon after a few seconds of inactivity
     }
 
     protected static String getBasicGradlePitestConfig() {
-        return """
```

Comment:
I like `return`s in methods.

Reply:
Me too! I'll disable UnnecessaryReturnKeyword rule.